src/include/parser/port_db.h

```diff
@@ -9,15 +10,22 @@ namespace parser {
 class ExportDB : public Statement {
 public:
     explicit ExportDB(std::string filePath)
-        : Statement{common::StatementType::EXPORT_DATABASE}, filePath{std::move(filePath)} {}
+        : Statement{common::StatementType::EXPORT_DATABASE}, filePath{std::move(filePath)} {
+        fileType = common::FileType::CSV;
+    }
 
     inline void setParsingOption(parsing_option_t options) { parsingOptions = std::move(options); }
+    inline void setFileType(std::string type) {
+        fileType = common::FileTypeUtils::fromString(type);
+    }
     inline const parsing_option_t& getParsingOptionsRef() const { return parsingOptions; }
     inline std::string getFilePath() const { return filePath; }
+    inline common::FileType getFileType() const { return fileType; }
 
 private:
     parsing_option_t parsingOptions;
     std::string filePath;
+    common::FileType fileType;
```

Comment:
You can just keep a readerConfig in this class. If so let's rename it to `BoundFileInfo`